tools/flang2/flang2exe/CMakeLists.txt

```diff
@@ -132,12 +138,16 @@ target_compile_options(flang2
   ${COMPILE_OPTS}
   )
 
+  
 target_link_libraries(flang2
   PRIVATE
   flangArgParser
-  ${FLANG_LIB_DIR}/scutil.a
-  -lm
+  scutil
   )
+  
+if (NOT MSVC)
+  target_link_libraries(flang2  m)
```

Comment:
Since flang2 is written in C++, it is probably not necessary to link libm explicitly.